README.md

```diff
@@ -72,7 +72,12 @@ reasonably POSIX-compliant system.
 Installation
 ------------
 
-For some Linux distributives deb and rpm packages are avalillable.
+### Package installation
+
+For some Linux distributives internal packages with `sqlite_fdw` are avalillable.
+
+- [sqlite_fdw_14 rpm](https://pkgs.org/download/sqlite_fdw_14(x86-64)) for CentOS 9, RHEL 9, Rocky Linux 9, AlmaLinux 9.
+- [sqlite_fdw code source](https://aur.archlinux.org/packages/sqlite_fdw) for Arch Linux.
```

Comment:
Please change "code source" to "source code".

Reply:
"Code source" is not the same as "source code". Source code is here, but "code source" is URL with this source code after some special normalisation for Arch Linux environment. How can I write about this in `README`? This code source look like as any URL of rpm or deb file with source code content, but in special Arch Linux format.

Reply:
Thanks for your explanation.
I think your provided URL is package details for Arch Linux.
How do you think about changing it to "[sqlite_fdw package](https://aur.archlinux.org/packages/sqlite_fdw) for Arch Linux."?

Reply:
"Package" is not fully correct word for Arch Linux, there is no file with content or source code inside. I think "git package" will normal.

Reply:
Changed in https://github.com/pgspider/sqlite_fdw/pull/59/commits/d92bd4fbf5c49d367396f7b05aabdbed19541a92

Reply:
Thanks. Confirmed.